src/components/WalletDropdown/AuthenticatedHeader.tsx

```diff
@@ -239,7 +239,7 @@ export default function AuthenticatedHeader({ account, openSettings }: { account
           <StatusIcon connection={connection} size={40} />
           {account && (
             <AccountNamesWrapper>
-              <ThemedText.SubHeader color="textPrimary" fontWeight={500}>
+              <ThemedText.SubHeader>
```

Comment:
the added lineheight stretches out the profile pic, moves the MM logo below the POAP, and adds a little too much space between the ens and address
Old:
![Screen Shot 2023-04-06 at 09 46 10 ](https://user-images.githubusercontent.com/11512321/230444091-2cb4473e-0ec2-43d4-ac53-636b07579df1.png)

New:
![Screen Shot 2023-04-06 at 09 46 06 ](https://user-images.githubusercontent.com/11512321/230444138-3050b7b5-4994-4afe-916c-71f2aeacd379.png)

Reply:
looks like i'll need to reduce the gap between those to 2px: 
<img width="704" alt="image" src="https://user-images.githubusercontent.com/59578595/230501622-eece7351-9d25-43be-88cc-eb8063f81e20.png">